nebu/cli/get.py

```diff
@@ -77,7 +79,11 @@ def get(ctx, env, col_id, col_version, output_dir, book_tree, get_resources):
 
     # ... and check if it's already been downloaded
     if output_dir.exists():
-        raise ExistingOutputDir(output_dir)
+        try:
+            raise ExistingOutputDir(output_dir.relative_to(Path.cwd()))
+        except ValueError:
+            # Raised ONLY when output_dir is not a child of cwd
+            raise ExistingOutputDir(output_dir)
```

Comment:
Another while-I'm-here - it bugged me that this was the only time reporting to the user that we used the full path, when it was relative (I hit this a lot when testing various `get` issues, when I'd forget to delete/move the last tree